client/src/components/HomePage/modal/JoinModal/index.jsx

```diff
@@ -33,14 +38,18 @@ class Join extends Component {
             context,
           );
         } else if (!validator.equals(password, confirmPassword)) {
-          this.setPopUp(
-            'Passwords do not match',
-            'Error !',
-            context,
-          );
+          this.setPopUp('Passwords do not match', 'Error !', context);
```

Comment:
typo: shouldn't be a space between the word and the exclamation mark.
```suggestion
          this.setPopUp('Passwords do not match', 'Error!', context);
```